source/conf.py

```diff
@@ -312,16 +253,16 @@ def setup(app):
 
 latex_elements = {
     "fontpkg": r"""
-	\setmainfont{DejaVu Serif}
```

Comment:
These got hit when I tried to do a tabs->spaces cleanup. It's not directly related to this activity but seems like worthwhile cleanup?